### ISSUE TITLE
Hyper-params for reprodcuing results of base reader in FiD

### ISSUE THREAD

**Issue by akhilkedia** [facebookresearch/FiD#16]
I tried reproducing the results of the base reader of FiD, but I am getting a test set score of 47.8 (i should be getting around 50).

I ran the below parameters on 8-gpus, and correspondingly to achieve the reported batch size of 64, I used 4 accumulation steps, and increased total_steps and eval_freq proportionally.

```bash
python -m torch.distributed.launch --nproc_per_node=8 --master-port 1234 train_reader.py --train_data  train.json --eval_data dev.json --per_gpu_batch_size 2 --accumulation_steps=4 --n_context 100 --checkpoint_dir $OUTPUT_DIR --name my_train --total_steps 40000 --text_maxlength 250 --eval_freq=2000 
```

Can you tell me which hyperparams to use to reproduce the base reader results for FiD?

**Comment by zhyunlong** [comment]
hi, do you figure out which hyperparams to use to reproduce the base reader results for FiD?

**Comment by YueChenkkk** [comment]
I got a dev rouge score of 22.4 on NarrativeQA, but it should be 29.9 as reported in their paper. I used huggingface evaluate to load the rouge metric, not sure whether they had used something else to calculate the metrics.

**Comment by Xnhyacinth** [comment]
> I tried reproducing the results of the base reader of FiD, but I am getting a test set score of 47.8 (i should be getting around 50).
> 
> I ran the below parameters on 8-gpus, and correspondingly to achieve the reported batch size of 64, I used 4 accumulation steps, and increased total_steps and eval_freq proportionally.
> 
> ```shell
> python -m torch.distributed.launch --nproc_per_node=8 --master-port 1234 train_reader.py --train_data  train.json --eval_data dev.json --per_gpu_batch_size 2 --accumulation_steps=4 --n_context 100 --checkpoint_dir $OUTPUT_DIR --name my_train --total_steps 40000 --text_maxlength 250 --eval_freq=2000 
> ```
> 
> Can you tell me which hyperparams to use to reproduce the base reader results for FiD?

So do you reproduce the base reader results for FiD? can you share the hyperparams ? Thanks!
